ubuntu-advantage.1

```diff
@@ -89,6 +89,39 @@ The possible values are \fIyes\fR or \fIno\fR.
 Show version of the Ubuntu Advantage package.
 
 
+.SH TIMER JOBS
+UA client sets up a systemd timer to run jobs that need to be executed
+recurrently. The timer itself ticks every 5 minutes on average, and decides
+which jobs need to be executed based on their intervals.
+
+Jobs are executed by the timer script if the script has not yet run
+successfully, or their interval since last successful run is already exceeded.
+There is a random delay applied to the timer, to desynchronize job execution
+time on machines spinned at the same time, avoiding multiple synchronized
+calls to the same service.
```

Comment:
Maybe this context is not needed on the manpage. What do you think @blackboxsw @orndorffgrant ?

Reply:
I like it. The more content the merrier. We can iterate and backoff on docs if we find they are getting stale, or don't belong in the commandline documentation